										
								</div>
							</section>

						<!-- Four -->
							<section id="four">
								<div class="container">
									<h3>Software</h3>
									
									All of my (recent) code is on <a href="https://github.com/davidaknowles/">github</a>, in particular: 
				<ul>
<li><a href="https://github.com/davidaknowles/leafcutter">LeafCutter RNA splicing tool</a>. 
<li><a href="https://github.com/davidaknowles/eagle">Detecting gene x environment interaction effects using allele-specific expression.</a>. 
				</ul>
					
Code from older projects:
				<ul>
<li>C#/Infer.NET code for Gaussian Process Regression Networks is <a href="https://github.com/davidaknowles/gprn">on github</a>. 
<li>Matlab code for nonparametric sparse factor analysis is available <a href="https://github.com/davidaknowles/nsfa">here</a>. 
<li>The MCMC sampler for Dirichlet Process Variable Clustering is at <a href="https://github.com/davidaknowles/dpvc">https://github.com/davidaknowles/dpvc</a>
				</ul>
								
								</div>
							</section>

							<section id="five">
								<div class="container">
									
										<h3>Reports/Abstracts/Presentations</h3>
										<nav id="nav_more">
										<ul class="feature-icons">
											<li class="fa-group"><a href="#abstracts">Workshop papers/conference abstracts</a></li>
											<li class="fa-paperclip"><a href="#reports">Reports/Theses</a></li>
											<li class="fa-video-camera"><a href="#presentations">Presentations</a></li>
										</nav>
										</ul>
							
						<section id="abstracts">			
						<h4>Workshop papers/conference abstracts</h4> 

						<UL>

						<li>Yang Li, Bryce van de Geijn, Allegra Petti, Anil Raj, <b>David A. Knowles</b>, John Blischak, Yoav Gilad, Jonathan Pritchard (2015).<br>
						<b>The effects of human genetic variation on the gene regulatory cascade.</b> <br>
						<i>American Society of Human Genetics 65th Annual Meeting</i> </li> 

						<li><b>David A. Knowles</b>, Stanley Ho, Kien Nguyen, Don Morris, Anthony Magliocco, Anindya Sarkar, Daphne Koller, Sylvia Plevritis, Srinivas Chukka, Michael Barnes (2015). <br>
						<b>Machine Learning-based Prognostication of Breast Cancer Recurrence using Tissue Slide Features. </b> <br>
						<i>Pathology Visions</i> Winner: Best Poster in Image Analysis!<br> </li>

						<li><b>David A. Knowles</b>, Joe R. Davis, Stephen B. Montgomery, Alexis Battle (2015).<br>
						<b>Detecting gene-by-environment interactions using allele specific expression.</b> <br>
						<i>The Biology of Genomes Meeting (CSHL) </i> <br> </li>

						<li><b>David A. Knowles</b>, Stanley Ho, Kien Nguyen, Don Morris, Anthony Magliocco, Anindya Sarkar, Daphne Koller, Srinivas Chukka, Michael Barnes (2014)<br>
						<b>Machine learning-based prognostication of breast cancer recurrence using tissue slide features from H&E and immunohistochemically stained slides.</b> <br> 
						<i>San Antonio Breast Cancer Symposium</i><br> </li>

						<li>Emily K. Tsang, Xin Li, Vanessa Anaya, Konrad J. Karczewski, <b>David A. Knowles</b>, Kevin S. Smith, Stepehn B. Montgomery (2014).<br>
						<b>Dissecting the genetic regulation of exosome RNA cargo in a large family.</b><br>
						<i>American Society of Human Genetics 64th Annual Meeting</i><br></li>

						<li>J.R. Davis, <b>D.A. Knowles</b>, S.B. Montgomery, A. Battle (2014) <br>
						<b>Rare variation and the genomic context of allele-specific expression.</b>
						<i>American Society of Human Genetics 64th Annual Meeting</i> <br> </li>

						<li>David A. Knowles, Alexis Battle, Daphne Koller (2013)<br>
						<b>Discovering latent cancer characteristics predictive of drug sensitivity.</b><br>
						<i>RECOMB/ISCB Conference on Regulatory & Systems Genomics (selected for oral presentation)</i></li>

						<li>Alexis Battle*, David A. Knowles*, Sara Mostafavi, Xiaowei Zhu, James B. Potash, Myrna M. Weissman, Courtney McCormick, Christian D. Haudenschild, Kenneth B. Beckman, Jianxin Shi, Rui Mei, Alexander E. Urban, Douglas F. Levinson, Daphne Koller, Stephen B. Montgomery (2013)<br>
						<b>The relationship between common environmental and genetic effects on human gene splicing and expression.</b><br>
						<i>American Society of Human Genetics (ASHG) Annual Meeting</i></li>

						<li> David A. Knowles, Leopold Parts, Daniel Glass and John M. Winn <br>
						<b>Inferring a measure of physiological age from multiple ageing related phenotypes.</b> <a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoParGlaWin11.pdf">paper</a> <a href="http://videolectures.net/nipsworkshops2011_knowles_phenotypes/">video</a> <br>
						To appear at the NIPS workshop: <a href="http://agbs.kyb.tuebingen.mpg.de/wikis/bg/NIPSPM11">From Statistical Genetics to Predictive Models in Personalized Medicine (NIPS PM 2011)</a> </li>

						<li> David A. Knowles, Leopold Parts, Daniel Glass and John M. Winn (2010) <br>
						<b>Modeling skin and ageing phenotypes using latent variable models in Infer.NET.</b> <a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoParGlaWin10.pdf">paper</a> <a href="knowles2010_predictive_medicine_poster.pdf">poster</a><br>
						Poster presented at: <a href="http://nips.cc/Conferences/2010/Program/event.php?ID=1979">Predictive Models in Personalized Medicine Workshop</a>, NIPS 2010, 6-11 December 2010, Vancouver, BC, Canada. </li>

						<li> Knowles, D. and Holmes, S. (2009) <br>
						<b>Statistical tools for ultra-deep pyrosequencing of fast evolving
						  viruses.</b> <a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoHol09.pdf">pdf</a> <a href="http://videolectures.net/nipsworkshops09_knowles_smud/">video</a>
						<a href="http://videolectures.net/site/normal_dl/tag=62979/nipsworkshops09_knowles_smud_01.pdf">slides</a><br>
						Presented at: Computational Biology Workshop, NIPS 2009, 7-12 December 2009, Vancouver, BC, Canada. </li>
						</UL>
</section>			

<section id="reports">			
						<h4>Reports/Theses</h4>

						<UL>

						<li> <b> Bayesian non-parametric models and inference for sparse and hierarchical latent structure (2012) <a href="documents/daknowles_thesis.pdf">pdf</a> </b>  <br>
						PhD Thesis, University of Cambridge<br> 
						Supervisor: Zoubin Ghahramani</li>

						<li> <b> Serial and Parallel Inference in Sparse Nonparametric Latent Factor Models applied to Gene Expression Modeling (2009) <a href="documents/knowles-fyr-2009.pdf">pdf</a> </b>  <br>
						PhD First Year Report, Department of Engineering, University of Cambridge<br> 
						Supervisor: Zoubin Ghahramani</li>

						<li> <b> Statistical tools for ulta-deep pyrosequencing of fast evolving viruses (2008) <a href="documents/454report.pdf">pdf</a> </b>  <br>
						MSc Bioinformatics and Systems Biology, Imperial College London, Individual Project <br> 
						Supervisor: Professor Susan Holmes, Stanford University</li>

						<li> <b> SBML-ABC: a package for data simulation, parameter inference and model selection, Group Report (2008) <a href="documents/abcsmc_group.pdf">pdf</a> </b>  <br>
						MSc Bioinformatics and Systems Biology, Imperial College London, Group Project<br> 
						Supervisor: Professor Michael Stumpf</li>

						<li> <b> Infinite Independent Components Analysis (2007) <a href="documents/iICA_report.pdf">pdf</a> </b>  <br>
						MEng Information Engineering, Cambridge University, 4th year project<br> 
						Supervisor: Professor Zoubin Ghahramani</li>

						<li> <b>Real Time Continuous Curvature Path Planner for an Autonomous Vehicle in an Urban Environment (2006) <a href="documents/knowles-surf06.pdf">pdf</a> </b><br>
						<a href="http://www.surf.caltech.edu/">Summer Undergraduate Research Fellowship</a>, Caltech. I was a member of <a href="http://gc.caltech.edu/public/Team_Caltech">Team Caltech</a>, an entry into the <a href="http://en.wikipedia.org/wiki/DARPA_Grand_Challenge_(2007)">DARPA Urban Challenge</a><br>
						Supervisor: Professor Richard Murray </li>
						</UL>
					</section>
						
<section id="presentations">	
						<h4>Presentations</h4>

						<UL>

						<li>Detecting gene-by-environment interactions using allele specific expression. <a href="http://meetings.cshl.edu/meetings/2015/genome15.shtml">The Biology of Genomes 2015</a> (image credit <a href="https://twitter.com/atjcagan">@AlexCagan</a>) <img src="davidk_bog.jpg" width=500>
						</li> 

						<li> <a href="documents/bnp_lecture.pdf">Properties of Bayesian nonparametric models and priors over trees</a>. Guest lecture as part of Matt Hoffman's STAT300 class, summer 2013. </li> 

						<li> <a href="documents/alberto.pdf">Diffusion trees as priors.</a> This was a talk I gave about the Dirichlet diffusion tree and Pitman Yor diffusion tree at Collegio Carlo Alberto. </li> 

						<li> <b>Inferring an individual's "physiological" age from multiple ageing-related phenotypes</b><br> I gave a talk at the Cambridge Statistics Initiative Special One-Day Meeting, which you can watch <a href="http://sms.cam.ac.uk/media/1177230">here</a>. I also presented this work at the NIPS 2011 Personalised Medicine workshop: <a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoParGlaWin11.pdf">paper</a> <a href="http://videolectures.net/nipsworkshops2011_knowles_phenotypes/">video</a> </li> 

						<li> <b>Variational methods for nonparametric Bayesian models</b><br> I gave a brief presentation at Microsoft Research summarising some attempts to use variational inference in nonparametric, particularly Dirichlet Process based, models. The slides are <a href="documents/msrpres.pdf">here</a>. </li> 

						</UL>
						</section>
							</div>
						</section>
						
						<section id="six">
														<div class="container">
															<h3>Miscellaneous</h3>
									
<p>I was involved with running the <a href="http://www.statslab.cam.ac.uk/clinic/">Cambridge University Statistics Clinic</a>. At Stanford I help out with <a href="http://stanford.edu/~rezab/smacc/">SMACC</a>: Statistical, Mathematical, and Computational Consulting. </p>

<h4>Lagrangian duality</h4>

<p>I gave journal club was on convex optimisation. I think I finally got my head round Lagrangian duality, and hopefully came up with a reasonably intuitive explanation. My focus is on intuition rather than rigor, and is based almost entirely on Boyd and Vandenberghe's <a href="http://www.stanford.edu/~boyd/cvxbook/">tome</a>. I thought I should write this up while it's still fresh, so here you go: <a href="lagrangian_duality.pdf">Lagrangian Duality for Dummies</a>

<h4>Binomial p-values</h4>

<p>Following my <a href="454report.pdf">work</a> on 454 pyrosequencing error rates with Professor Holmes, I was asked about how to calculate a p-value for comparing two draws from a Binomial distribution to test the hypothesis that the number of substitutions seen in the sample is significantly greater than the number of substitutions seen in the control. There is actually no need to use the Poisson approximation, and the Binomial distribution very naturally takes care of varying coverage. I explain my approach <a href="documents/454pvalues.pdf">here</a>. </p>

<h4>Infer.NET</h4>

<p>During my PhD I was a  part-time developer of <a href="http://research.microsoft.com/en-us/um/cambridge/projects/infernet/">Infer.NET</a>. I wrote a blog post about some of the features we added in Infer.NET 2.4, see <a href="http://blogs.msdn.com/b/infernet_team_blog/archive/2011/09/30/new-features-in-infer-net-2-4-the-softmax-factor.aspx">here</a>. </p>

									
														</div>
													</section>
						


					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>&copy; Untitled. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollzer.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>
<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>
	</body>
</html>
