<ol>
<li><div id="Knowles2017gxe"><em>Knowles DA, Davis JR, Edgington H, Raj A, Fav&eacute; M-J, Zhu X, Potash JB, Weissman MM, Shi J, Levinson D, Awadalla P, Mostafavi S, Montgomery SB and Battle A</em> (2017), <strong>"Allele-specific expression reveals interactions between genetic variation and environment"</strong>, Nature Methods. </div>
<div>[<a data-toggle="collapse" href="#abs_Knowles2017gxe">Abstract</a>][<a href="#bib_Knowles2017gxe" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/nmeth.4298" target="_blank">DOI</a>] [<a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4298.html" target="_blank">URL</a>] [<a href="http://biorxiv.org/content/early/2015/09/13/025874.abstract" target="_blank">Preprint</a>] [<a href="https://github.com/davidaknowles/eagle" target="_blank">Code</a>]</div>

<div id="abs_Knowles2017gxe" class="collapse">
	<b>Abstract</b>: Identifying interactions between genetics and the environment (GxE) remains challenging. We have developed EAGLE, a hierarchical Bayesian model for identifying GxE interactions based on association between environment and allele-specific expression (ASE). Combining RNA-sequencing of whole blood and extensive environmental annotations collected from 922 human individuals, we identified 35 GxE interactions, compared to only four using standard GxE testing. EAGLE provides new opportunities to identify GxE interactions using functional genomic data.
</div>
<div id="bib_Knowles2017gxe" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2017gxe,
  author = {Knowles, David A and Davis, Joe R and Edgington, Hilary and Raj, Anil and Fav&eacute;, Marie-Julie and Zhu, Xiaowei and Potash, James B and Weissman, Myrna M and Shi, Jianxin and Levinson, Doug and Awadalla, Philip and Mostafavi, Sara and Montgomery, Stephen B and Battle, Alexis},
  title = {Allele-specific expression reveals interactions between genetic variation and environment},
  journal = {Nature Methods},
  year = {2017},
  url = {http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4298.html},
  doi = {10.1038/nmeth.4298}
}
</pre>
</div>
</li><li><div id="Tung2017"><em>Tung P-Y, Blischak JD, Hsiao CJ, Knowles DA, Burnett JE, Pritchard JK and Gilad Y</em> (2017), <strong>"Batch effects and the effective design of single-cell gene expression studies"</strong>, Scientific Reports.  Vol. 7, pp. 39921.</div>
<div>[<a data-toggle="collapse" href="#abs_Tung2017">Abstract</a>][<a href="#bib_Tung2017" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/srep39921" target="_blank">DOI</a>] [<a href="http://www.nature.com/articles/srep39921" target="_blank">URL</a>]</div>

<div id="abs_Tung2017" class="collapse">
	<b>Abstract</b>: Single cell RNA sequencing (scRNA-seq) can be used to characterize variation in gene expression levels at high resolution. However, the sources of experimental noise in scRNA-seq are not yet well understood. We investigated the technical variation associated with sample processing using the single cell Fluidigm C1 platform. To do so, we processed three C1 replicates from three human induced pluripotent stem cell (iPSC) lines. We added unique molecular identifiers (UMIs) to all samples, to account for amplification bias. We found that the major source of variation in the gene expression data was driven by genotype, but we also observed substantial variation between the technical replicates. We observed that the conversion of reads to molecules using the UMIs was impacted by both biological and technical variation, indicating that UMI counts are not an unbiased estimator of gene expression levels. Based on our results, we suggest a framework for effective scRNA-seq studies.
</div>
<div id="bib_Tung2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Tung2017,
  author = {Tung, Po-Yuan and Blischak, John D. and Hsiao, Chiaowen Joyce and Knowles, David A. and Burnett, Jonathan E. and Pritchard, Jonathan K. and Gilad, Yoav},
  title = {Batch effects and the effective design of single-cell gene expression studies},
  journal = {Scientific Reports},
  year = {2017},
  volume = {7},
  pages = {39921},
  url = {http://www.nature.com/articles/srep39921},
  doi = {10.1038/srep39921}
}
</pre>
</div>
</li><li><div id="Calderon2017"><em>Calderon D, Bhaskar A, Knowles DA, Golan D, Raj T, Fu A and Pritchard JK</em> (2017), <strong>"Inferring Relevant Cell Types For Complex Traits Using Single-Cell Gene Expression"</strong>, The American Journal of Human Genetics. , pp. 136283. *To appear..</div>
<div>[<a data-toggle="collapse" href="#abs_Calderon2017">Abstract</a>][<a href="#bib_Calderon2017" data-toggle="collapse">BibTeX</a>] [<a href="http://biorxiv.org/content/early/2017/05/10/136283.abstract" target="_blank">URL</a>] [<a href="https://cran.r-project.org/web/packages/rolypoly/index.html" target="_blank">Code</a>]</div>

<div id="abs_Calderon2017" class="collapse">
	<b>Abstract</b>: Previous studies have prioritized trait-relevant cell types by looking for an enrichment of GWAS signal within functional regions. However, these studies are limited in cell resolution by the lack of functional annotations from difficult-to-characterize or rare cell populations. Measurement of single-cell gene expression has become a popular method for characterizing novel cell types, and yet, hardly any work exists linking single-cell RNA-seq to phenotypes of interest. To address this deficiency, we present RolyPoly, a regression-based polygenic model that can prioritize trait-relevant cell types and genes from GWAS summary statistics and single-cell RNA-seq. We demonstrate RolyPoly's accuracy through simulation and validate previously known tissue-trait associations. We discover a significant association between microglia and late-onset Alzheimer's disease, and an association between oligodendrocytes and replicating fetal cortical cells with schizophrenia. Additionally, RolyPoly computes a trait-relevance score for each gene which reflects the importance of expression specific to a cell type. We found that differentially expressed genes in the prefrontal cortex of Alzheimer's patients were significantly enriched for highly ranked genes by RolyPoly gene scores. Overall, our method represents a powerful framework for understanding the effect of common variants on cell types contributing to complex traits.
</div>
<div id="bib_Calderon2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Calderon2017,
  author = {Calderon, Diego and Bhaskar, Anand and Knowles, David A and Golan, David and Raj, Towfique and Fu, Audrey and Pritchard, Jonathan K},
  title = {Inferring Relevant Cell Types For Complex Traits Using Single-Cell Gene Expression},
  journal = {The American Journal of Human Genetics},
  year = {2017},
  pages = {136283},
  url = {http://biorxiv.org/content/early/2017/05/10/136283.abstract}
}
</pre>
</div>
</li><li><div id="Tsang2017"><em>Tsang EK, Abell NS, Li X, Anaya V, Karczewski KJ, Knowles DA, Sierra RG, Smith KS and Montgomery SB</em> (2017), <strong>"Small RNA sequencing in cells and exosomes identifies eQTLs and 14q32 as a region of active export"</strong>, G3 Genes|Genomes|Genetics.  Vol. 7(1), pp. 31-39.</div>
<div>[<a data-toggle="collapse" href="#abs_Tsang2017">Abstract</a>][<a href="#bib_Tsang2017" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1534/g3.116.036137" target="_blank">DOI</a>] [<a href="http://g3journal.org/lookup/doi/10.1534/g3.116.036137" target="_blank">URL</a>]</div>

<div id="abs_Tsang2017" class="collapse">
	<b>Abstract</b>: Exosomes are small extracellular vesicles that carry heterogeneous cargo, including RNA, between cells. Increasing evidence suggests that exosomes are important mediators of intercellular communication and biomarkers of disease. Despite this, the variability of exosomal RNA between individuals has not been well quantified. To assess this variability, we sequenced the small RNA of cells and exosomes from a 17-member family. Across individuals, we show that selective export of miRNAs occurs not only at the level of specific transcripts, but that a cluster of 74 mature miRNAs on chromosome 14q32 is massively exported in exosomes while mostly absent from cells. We also observe more interindividual variability between exosomal samples than between cellular ones and identify four miRNA expression quantitative trait loci shared between cells and exosomes. Our findings indicate that genomically colocated miRNAs can be exported together and highlight the variability in exosomal miRNA levels between individuals as relevant for exosome use as diagnostics.
</div>
<div id="bib_Tsang2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Tsang2017,
  author = {Tsang, Emily K. and Abell, Nathan S. and Li, Xin and Anaya, Vanessa and Karczewski, Konrad J. and Knowles, David A. and Sierra, Raymond G. and Smith, Kevin S. and Montgomery, Stephen B.},
  title = {Small RNA sequencing in cells and exosomes identifies eQTLs and 14q32 as a region of active export},
  journal = {G3 Genes|Genomes|Genetics},
  year = {2017},
  volume = {7},
  number = {1},
  pages = {31--39},
  url = {http://g3journal.org/lookup/doi/10.1534/g3.116.036137},
  doi = {10.1534/g3.116.036137}
}
</pre>
</div>
</li><li><div id="Becker2017"><em>Becker LA, Huang B, Bieri G, Ma R, Knowles DA, Jafar-Nejad P, Messing J, Kim HJ, Soriano A, Auburger G, Pulst SM, Taylor JP, Rigo F and Gitler AD</em> (2017), <strong>"Therapeutic reduction of ataxin-2 extends lifespan and reduces pathology in TDP-43 mice"</strong>, Nature.  Vol. 544(7650), pp. 367-371.</div>
<div>[<a data-toggle="collapse" href="#abs_Becker2017">Abstract</a>][<a href="#bib_Becker2017" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/nature22038" target="_blank">DOI</a>] [<a href="http://www.nature.com/doifinder/10.1038/nature22038" target="_blank">URL</a>]</div>

<div id="abs_Becker2017" class="collapse">
	<b>Abstract</b>: Amyotrophic lateral sclerosis (ALS) is a rapidly progressing neurodegenerative disease that is characterized by motor neuron loss and that leads to paralysis and death 2--5 years after disease onset 1 . Nearly all patients with ALS have aggregates of the RNA-binding protein TDP-43 in their brains and spinal cords 2 , and rare mutations in the gene encoding TDP-43 can cause ALS 3 . There are no effective TDP-43-directed therapies for ALS or related TDP-43 proteinopathies, such as frontotemporal dementia. Antisense oligonucleotides (ASOs) and RNA-interference approaches are emerging as attractive therapeutic strategies in neurological diseases 4 . Indeed, treatment of a rat model of inherited ALS (caused by a mutation in Sod1) with ASOs against Sod1 has been shown to substantially slow disease progression 5 . However, as SOD1 mutations account for only around 2--5&percnt; of ALS cases, additional therapeutic strategies are needed. Silencing TDP-43 itself is probably not appropriate, given its critical cellular functions 1,6 . Here we present a promising alternative therapeutic strategy for ALS that involves targeting ataxin-2. A decrease in ataxin-2 suppresses TDP-43 toxicity in yeast and flies 7 , and intermediate-length polyglutamine expansions in the ataxin-2 gene increase risk of ALS 7,8 . We used two independent approaches to test whether decreasing ataxin-2 levels could mitigate disease in a mouse model of TDP-43 proteinopathy 9 . First, we crossed ataxin-2 knockout mice with TDP-43 (also known as TARDBP) transgenic mice. The decrease in ataxin-2 reduced aggregation of TDP-43, markedly increased survival and improved motor function. Second, in a more therapeutically applicable approach, we administered ASOs targeting ataxin-2 to the central nervous system of TDP-43 transgenic mice. This single treatment markedly extended survival. Because TDP-43 aggregation is a component of nearly all cases of ALS 6 , targeting ataxin-2 could represent a broadly effective therapeutic strategy. To test the hypothesis that a decrease in ataxin-2 levels can res-cue neurodegenerative phenotypes caused by TDP-43 accumula-tion, we first used a genetic approach. There are several transgenic mouse lines that express wild-type or mutant TDP-43, using various strategies 10
</div>
<div id="bib_Becker2017" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Becker2017,
  author = {Becker, Lindsay A. and Huang, Brenda and Bieri, Gregor and Ma, Rosanna and Knowles, David A. and Jafar-Nejad, Paymaan and Messing, James and Kim, Hong Joo and Soriano, Armand and Auburger, Georg and Pulst, Stefan M. and Taylor, J. Paul and Rigo, Frank and Gitler, Aaron D.},
  title = {Therapeutic reduction of ataxin-2 extends lifespan and reduces pathology in TDP-43 mice},
  journal = {Nature},
  year = {2017},
  volume = {544},
  number = {7650},
  pages = {367--371},
  url = {http://www.nature.com/doifinder/10.1038/nature22038},
  doi = {10.1038/nature22038}
}
</pre>
</div>
</li><li><div id="Davis2016eigenmt"><em>Davis JR, Fresard L, Knowles DA, Pala M, Bustamante CD, Battle A and Montgomery SB</em> (2016), <strong>"An Efficient Multiple-Testing Adjustment for eQTL Studies that Accounts for Linkage Disequilibrium between Variants"</strong>, The American Journal of Human Genetics.  Vol. 98(1), pp. 216-224.</div>
<div>[<a data-toggle="collapse" href="#abs_Davis2016eigenmt">Abstract</a>][<a href="#bib_Davis2016eigenmt" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1016/j.ajhg.2015.11.021" target="_blank">DOI</a>] [<a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00492-9" target="_blank">URL</a>] [<a href="http://montgomerylab.stanford.edu/resources/eigenMT/eigenMT.html" target="_blank">Code</a>]</div>

<div id="abs_Davis2016eigenmt" class="collapse">
	<b>Abstract</b>: Methods for multiple-testing correction in local expression quantitative trait locus (cis-eQTL) studies are a trade-off between statistical power and computational efficiency. Bonferroni correction, though computationally trivial, is overly conservative and fails to account for linkage disequilibrium between variants. Permutation-based methods are more powerful, though computationally far more intensive. We present an alternative correction method called eigenMT, which runs over 500 times faster than permutations and has adjusted p values that closely approximate empirical ones. To achieve this speed while also maintaining the accuracy of permutation-based methods, we estimate the effective number of independent variants tested for association with a particular gene, termed Meff, by using the eigenvalue decomposition of the genotype correlation matrix. We employ a regularized estimator of the correlation matrix to ensure Meff is robust and yields adjusted p values that closely approximate p values from permutations. Finally, using a common genotype matrix, we show that eigenMT can be applied with even greater efficiency to studies across tissues or conditions. Our method provides a simpler, more efficient approach to multiple-testing correction than existing methods and fits within existing pipelines for eQTL discovery.
</div>
<div id="bib_Davis2016eigenmt" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Davis2016eigenmt,
  author = {Davis, Joe R. and Fresard, Laure and Knowles, David A. and Pala, Mauro and Bustamante, Carlos D. and Battle, Alexis and Montgomery, Stephen B.},
  title = {An Efficient Multiple-Testing Adjustment for eQTL Studies that Accounts for Linkage Disequilibrium between Variants},
  journal = {The American Journal of Human Genetics},
  year = {2016},
  volume = {98},
  number = {1},
  pages = {216--224},
  url = {http://www.cell.com/ajhg/abstract/S0002-9297(15)00492-9},
  doi = {10.1016/j.ajhg.2015.11.021}
}
</pre>
</div>
</li><li><div id="Kukurba2015"><em>Kukurba KR, Parsana P, Balliu B, Smith KS, Zappala Z, Knowles DA, Fav&eacute; M-J, Davis JR, Li X, Zhu X, Potash JB, Weissman MM, Shi J, Kundaje A, Levinson DF, Awadalla P, Mostafavi S, Battle A and Montgomery SB</em> (2016), <strong>"Impact of the X chromosome and sex on regulatory variation"</strong>, Genome Research.  Vol. 26(6), pp. 768-777.</div>
<div>[<a data-toggle="collapse" href="#abs_Kukurba2015">Abstract</a>][<a href="#bib_Kukurba2015" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1101/gr.197897.115" target="_blank">DOI</a>] [<a href="http://genome.cshlp.org/lookup/doi/10.1101/gr.197897.115" target="_blank">URL</a>]</div>

<div id="abs_Kukurba2015" class="collapse">
	<b>Abstract</b>: The X chromosome, with its unique mode of inheritance, contributes to differences between the sexes at a molecular level, including sex-specific gene expression and sex-specific impact of genetic variation. We have conducted an analysis of the impact of both sex and the X chromosome on patterns of gene expression identified through transcriptome sequencing of whole blood from 922 individuals. We identified that genes on the X chromosome are more likely to have sex-specific expression compared to the autosomal genes. Furthermore, we identified a depletion of regulatory variants on the X chromosome, especially among genes under high selective constraint. In contrast, we discovered an enrichment of sex-specific regulatory variants on the X chromosome. To resolve the molecular mechanisms underlying such effects, we generated and connected sex-specific chromatin accessibility to sex-specific expression and regulatory variation. As sex-specific regulatory variants can inform sex differences in genetic disease prevalence, we have integrated our data with genome-wide association study data for multiple immune traits and to identify traits with significant sex biases. Together, our study provides genome-wide insight into how the X chromosome and sex shape human gene regulation and disease.
</div>
<div id="bib_Kukurba2015" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Kukurba2015,
  author = {Kukurba, Kimberly R. and Parsana, Princy and Balliu, Brunilda and Smith, Kevin S. and Zappala, Zachary and Knowles, David A. and Fav&eacute;, Marie-Julie and Davis, Joe R. and Li, Xin and Zhu, Xiaowei and Potash, James B. and Weissman, Myrna M. and Shi, Jianxin and Kundaje, Anshul and Levinson, Douglas F. and Awadalla, Philip and Mostafavi, Sara and Battle, Alexis and Montgomery, Stephen B.},
  title = {Impact of the X chromosome and sex on regulatory variation},
  journal = {Genome Research},
  publisher = {Cold Spring Harbor Labs Journals},
  year = {2016},
  volume = {26},
  number = {6},
  pages = {768--777},
  url = {http://genome.cshlp.org/lookup/doi/10.1101/gr.197897.115},
  doi = {10.1101/gr.197897.115}
}
</pre>
</div>
</li><li><div id="Li2016splicing"><em>Li YI, van de Geijn B, Raj A, Knowles DA, Petti AA, Golan D, Gilad Y and Pritchard JK</em> (2016), <strong>"RNA splicing is a primary link between genetic variation and disease."</strong>, Science.  Vol. 352(6285), pp. 600-4.</div>
<div>[<a data-toggle="collapse" href="#abs_Li2016splicing">Abstract</a>][<a href="#bib_Li2016splicing" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1126/science.aad9417" target="_blank">DOI</a>] [<a href="http://www.ncbi.nlm.nih.gov/pubmed/27126046" target="_blank">URL</a>]</div>

<div id="abs_Li2016splicing" class="collapse">
	<b>Abstract</b>: Noncoding variants play a central role in the genetics of complex traits, but we still lack a full understanding of the molecular pathways through which they act. We quantified the contribution of cis-acting genetic effects at all major stages of gene regulation from chromatin to proteins, in Yoruba lymphoblastoid cell lines (LCLs). About 65&percnt; of expression quantitative trait loci (eQTLs) have primary effects on chromatin, whereas the remaining eQTLs are enriched in transcribed regions. Using a novel method, we also detected 2893 splicing QTLs, most of which have little or no effect on gene-level expression. These splicing QTLs are major contributors to complex traits, roughly on a par with variants that affect gene expression levels. Our study provides a comprehensive view of the mechanisms linking genetic variation to variation in human gene regulation.
</div>
<div id="bib_Li2016splicing" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Li2016splicing,
  author = {Li, Yang I and van de Geijn, Bryce and Raj, Anil and Knowles, David A and Petti, Allegra A and Golan, David and Gilad, Yoav and Pritchard, Jonathan K},
  title = {RNA splicing is a primary link between genetic variation and disease.},
  journal = {Science},
  publisher = {American Association for the Advancement of Science},
  year = {2016},
  volume = {352},
  number = {6285},
  pages = {600--4},
  url = {http://www.ncbi.nlm.nih.gov/pubmed/27126046},
  doi = {10.1126/science.aad9417}
}
</pre>
</div>
</li><li><div id="Kukurba2014deleterious"><em>Kukurba KR, Zhang R, Li X, Smith KS, Knowles DA, How Tan M, Piskol R, Lek M, Snyder M, MacArthur DG, Li JB and Montgomery SB</em> (2014), <strong>"Allelic Expression of Deleterious Protein-Coding Variants across Human Tissues"</strong>, PLoS Genetics.  Vol. 10(5), pp. e1004304.</div>
<div>[<a data-toggle="collapse" href="#abs_Kukurba2014deleterious">Abstract</a>][<a href="#bib_Kukurba2014deleterious" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1371/journal.pgen.1004304" target="_blank">DOI</a>] [<a href="http://dx.plos.org/10.1371/journal.pgen.1004304" target="_blank">URL</a>]</div>

<div id="abs_Kukurba2014deleterious" class="collapse">
	<b>Abstract</b>: Personal exome and genome sequencing provides access to loss-of-function and rare deleterious alleles whose interpretation is expected to provide insight into individual disease burden. However, for each allele, accurate interpretation of its effect will depend on both its penetrance and the trait's expressivity. In this regard, an important factor that can modify the effect of a pathogenic coding allele is its level of expression; a factor which itself characteristically changes across tissues. To better inform the degree to which pathogenic alleles can be modified by expression level across multiple tissues, we have conducted exome, RNA and deep, targeted allele-specific expression (ASE) sequencing in ten tissues obtained from a single individual. By combining such data, we report the impact of rare and common loss-of-function variants on allelic expression exposing stronger allelic bias for rare stop-gain variants and informing the extent to which rare deleterious coding alleles are consistently expressed across tissues. This study demonstrates the potential importance of transcriptome data to the interpretation of pathogenic protein-coding variants.
</div>
<div id="bib_Kukurba2014deleterious" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Kukurba2014deleterious,
  author = {Kukurba, Kimberly R. and Zhang, Rui and Li, Xin and Smith, Kevin S. and Knowles, David A. and How Tan, Meng and Piskol, Robert and Lek, Monkol and Snyder, Michael and MacArthur, Daniel G. and Li, Jin Billy and Montgomery, Stephen B.},
  title = {Allelic Expression of Deleterious Protein-Coding Variants across Human Tissues},
  journal = {PLoS Genetics},
  publisher = {Public Library of Science},
  year = {2014},
  volume = {10},
  number = {5},
  pages = {e1004304},
  url = {http://dx.plos.org/10.1371/journal.pgen.1004304},
  doi = {10.1371/journal.pgen.1004304}
}
</pre>
</div>
</li><li><div id="Li2014rare"><em>Li X, Battle A, Karczewski KJ, Zappala Z, Knowles DA, Smith KS, Kukurba KR, Wu E, Simon N and Montgomery SB</em> (2014), <strong>"Transcriptome sequencing of a large human family identifies the impact of rare noncoding variants."</strong>, American Journal of Human Genetics.  Vol. 95(3), pp. 245-56.</div>
<div>[<a data-toggle="collapse" href="#abs_Li2014rare">Abstract</a>][<a href="#bib_Li2014rare" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1016/j.ajhg.2014.08.004" target="_blank">DOI</a>] [<a href="http://www.cell.com/article/S0002929714003486/fulltext" target="_blank">URL</a>]</div>

<div id="abs_Li2014rare" class="collapse">
	<b>Abstract</b>: Recent and rapid human population growth has led to an excess of rare genetic variants that are expected to contribute to an individual's genetic burden of disease risk. To date, much of the focus has been on rare protein-coding variants, for which potential impact can be estimated from the genetic code, but determining the impact of rare noncoding variants has been more challenging. To improve our understanding of such variants, we combined high-quality genome sequencing and RNA sequencing data from a 17-individual, three-generation family to contrast expression quantitative trait loci (eQTLs) and splicing quantitative trait loci (sQTLs) within this family to eQTLs and sQTLs within a population sample. Using this design, we found that eQTLs and sQTLs with large effects in the family were enriched with rare regulatory and splicing variants (minor allele frequency 0.01). They were also more likely to influence essential genes and genes involved in complex disease. In addition, we tested the capacity of diverse noncoding annotation to predict the impact of rare noncoding variants. We found that distance to the transcription start site, evolutionary constraint, and epigenetic annotation were considerably more informative for predicting the impact of rare variants than for predicting the impact of common variants. These results highlight that rare noncoding variants are important contributors to individual gene-expression profiles and further demonstrate a significant capability for genomic annotation to predict the impact of rare noncoding variants..
</div>
<div id="bib_Li2014rare" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Li2014rare,
  author = {Li, Xin and Battle, Alexis and Karczewski, Konrad J. and Zappala, Zach and Knowles, David A. and Smith, Kevin S. and Kukurba, Kim R. and Wu, Eric and Simon, Noah and Montgomery, Stephen B.},
  title = {Transcriptome sequencing of a large human family identifies the impact of rare noncoding variants.},
  journal = {American Journal of Human Genetics},
  publisher = {Elsevier},
  year = {2014},
  volume = {95},
  number = {3},
  pages = {245--56},
  url = {http://www.cell.com/article/S0002929714003486/fulltext},
  doi = {10.1016/j.ajhg.2014.08.004}
}
</pre>
</div>
</li><li><div id="Glass2013muther"><em>Glass D, Vi&ntilde;uela A, Davies MN, Ramasamy A, Parts L, Knowles DA, Brown AA, Hedman AK, Small KS, Buil A, Grundberg E, Nica AC, Meglio P, Nestle FO, Ryten M, Durbin R, McCarthy MI, Deloukas P, Dermitzakis ET, Weale ME, Bataille V and Spector TD</em> (2013), <strong>"Gene expression changes with age in skin, adipose tissue, blood and brain."</strong>, Genome biology.  Vol. 14(7), pp. R75.</div>
<div>[<a data-toggle="collapse" href="#abs_Glass2013muther">Abstract</a>][<a href="#bib_Glass2013muther" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1186/gb-2013-14-7-r75" target="_blank">DOI</a>] [<a href="http://genomebiology.com/2013/14/7/R75" target="_blank">URL</a>]</div>

<div id="abs_Glass2013muther" class="collapse">
	<b>Abstract</b>: BACKGROUND: Previous studies have demonstrated that gene expression levels change with age. These changes are hypothesized to influence the aging rate of an individual. We analyzed gene expression changes with age in abdominal skin, subcutaneous adipose tissue and lymphoblastoid cell lines in 856 female twins in the age range of 39-85 years. Additionally, we investigated genotypic variants involved in genotype-by-age interactions to understand how the genomic regulation of gene expression alters with age.  RESULTS: Using a linear mixed model, differential expression with age was identified in 1,672 genes in skin and 188 genes in adipose tissue. Only two genes expressed in lymphoblastoid cell lines showed significant changes with age. Genes significantly regulated by age were compared with expression profiles in 10 brain regions from 100 postmortem brains aged 16 to 83 years. We identified only one age-related gene common to the three tissues. There were 12 genes that showed differential expression with age in both skin and brain tissue and three common to adipose and brain tissues.  CONCLUSIONS: Skin showed the most age-related gene expression changes of all the tissues investigated, with many of the genes being previously implicated in fatty acid metabolism, mitochondrial activity, cancer and splicing. A significant proportion of age-related changes in gene expression appear to be tissue-specific with only a few genes sharing an age effect in expression across tissues. More research is needed to improve our understanding of the genetic influences on aging and the relationship with age-related diseases.
</div>
<div id="bib_Glass2013muther" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Glass2013muther,
  author = {Glass, Daniel and Vi&ntilde;uela, Ana and Davies, Matthew N and Ramasamy, Adaikalavan and Parts, Leopold and Knowles, David A. and Brown, Andrew A and Hedman, Asa K and Small, Kerrin S and Buil, Alfonso and Grundberg, Elin and Nica, Alexandra C and Meglio, Paoladi and Nestle, Frank O and Ryten, Mina and Durbin, Richard and McCarthy, Mark I and Deloukas, Panagiotis and Dermitzakis, Emmanouil T and Weale, Michael E and Bataille, Veronique and Spector, Tim D},
  title = {Gene expression changes with age in skin, adipose tissue, blood and brain.},
  journal = {Genome biology},
  year = {2013},
  volume = {14},
  number = {7},
  pages = {R75},
  url = {http://genomebiology.com/2013/14/7/R75},
  doi = {10.1186/gb-2013-14-7-r75}
}
</pre>
</div>
</li><li><div id="Grundberg2012"><em>Grundberg E, Small KS, Hedman AK, Nica AC, Buil A, Keildson S, Bell JT, Yang T-P, Meduri E, Barrett A, Nisbett J, Sekowska M, Wilk A, Shin S-Y, Glass D, Travers M, Min JL, Knowles DA, Ring S, Ho K, Thorleifsson G, Kong A, Thorsteindottir U, Ainali C, Dimas AS, Hassanali N, Ingle C, Krestyaninova M, Lowe CE, Di Meglio P, Montgomery SB, Parts L, Potter S, Surdulescu G, Tsaprouni L, Tsoka S, Bataille V, Durbin R, Nestle FO, O'Rahilly S, Soranzo N, Lindgren CM, Zondervan KT, Ahmadi KR, Schadt EE, Stefansson K, Smith GD, McCarthy MI, Deloukas P, Dermitzakis ET and Spector TD</em> (2012), <strong>"Mapping cis- and trans-regulatory effects across multiple tissues in twins."</strong>, Nature Genetics.  Vol. 44(10), pp. 1084-9.</div>
<div>[<a data-toggle="collapse" href="#abs_Grundberg2012">Abstract</a>][<a href="#bib_Grundberg2012" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1038/ng.2394" target="_blank">DOI</a>] [<a href="http://dx.doi.org/10.1038/ng.2394" target="_blank">URL</a>]</div>

<div id="abs_Grundberg2012" class="collapse">
	<b>Abstract</b>: Sequence-based variation in gene expression is a key driver of disease risk. Common variants regulating expression in cis have been mapped in many expression quantitative trait locus (eQTL) studies, typically in single tissues from unrelated individuals. Here, we present a comprehensive analysis of gene expression across multiple tissues conducted in a large set of mono- and dizygotic twins that allows systematic dissection of genetic (cis and trans) and non-genetic effects on gene expression. Using identity-by-descent estimates, we show that at least 40&percnt; of the total heritable cis effect on expression cannot be accounted for by common cis variants, a finding that reveals the contribution of low-frequency and rare regulatory variants with respect to both transcriptional regulation and complex trait susceptibility. We show that a substantial proportion of gene expression heritability is trans to the structural gene, and we identify several replicating trans variants that act predominantly in a tissue-restricted manner and may regulate the transcription of many genes.
</div>
<div id="bib_Grundberg2012" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Grundberg2012,
  author = {Grundberg, Elin and Small, Kerrin S and Hedman, Asa K and Nica, Alexandra C and Buil, Alfonso and Keildson, Sarah and Bell, Jordana T and Yang, Tsun-Po and Meduri, Eshwar and Barrett, Amy and Nisbett, James and Sekowska, Magdalena and Wilk, Alicja and Shin, So-Youn and Glass, Daniel and Travers, Mary and Min, Josine L and Knowles, David A. and Ring, Sue and Ho, Karen and Thorleifsson, Gudmar and Kong, Augustine and Thorsteindottir, Unnur and Ainali, Chrysanthi and Dimas, Antigone S and Hassanali, Neelam and Ingle, Catherine and Krestyaninova, Maria and Lowe, Christopher E and Di Meglio, Paola and Montgomery, Stephen B and Parts, Leopold and Potter, Simon and Surdulescu, Gabriela and Tsaprouni, Loukia and Tsoka, Sophia and Bataille, Veronique and Durbin, Richard and Nestle, Frank O and O'Rahilly, Stephen and Soranzo, Nicole and Lindgren, Cecilia M and Zondervan, Krina T and Ahmadi, Kourosh R and Schadt, Eric E and Stefansson, Kari and Smith, George Davey and McCarthy, Mark I and Deloukas, Panos and Dermitzakis, Emmanouil T and Spector, Tim D.},
  title = {Mapping cis- and trans-regulatory effects across multiple tissues in twins.},
  journal = {Nature Genetics},
  year = {2012},
  volume = {44},
  number = {10},
  pages = {1084--9},
  url = {http://dx.doi.org/10.1038/ng.2394},
  doi = {10.1038/ng.2394}
}
</pre>
</div>
</li><li><div id="Schone2011"><em>Sch&ouml;ne C, Venner A, Knowles DA, Karnani MM and Burdakov D</em> (2011), <strong>"Dichotomous cellular properties of mouse orexin/hypocretin neurons."</strong>, The Journal of Physiology.  Vol. 589(Pt 11), pp. 2767-79.</div>
<div>[<a data-toggle="collapse" href="#abs_Schone2011">Abstract</a>][<a href="#bib_Schone2011" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1113/jphysiol.2011.208637" target="_blank">DOI</a>] [<a href="http://jp.physoc.org/content/early/2011/04/11/jphysiol.2011.208637.abstract" target="_blank">URL</a>]</div>

<div id="abs_Schone2011" class="collapse">
	<b>Abstract</b>: Hypothalamic hypocretin/orexin (Hcrt/Orx) neurons recently emerged as critical regulators of sleep--wake cycles, reward seeking and body energy balance. However, at the level of cellular and network properties, it remains unclear whether Hcrt/Orx neurons are one homogeneous population, or whether there are several distinct types of Hcrt/Orx cells. Here, we collated diverse structural and functional information about individual Hcrt/Orx neurons in mouse brain slices, by combining patch-clamp analysis of spike firing, membrane currents and synaptic inputs with confocal imaging of cell shape and subsequent 3-dimensional Sholl analysis of dendritic architecture. Statistical cluster analysis of intrinsic firing properties revealed that Hcrt/Orx neurons fall into two distinct types. These two cell types also differ in the complexity of their dendritic arbour, the strength of AMPA and GABAA receptor-mediated synaptic drive that they receive, and the density of low-threshold, 4-aminopyridine-sensitive, transient K+ current. Our results provide quantitative evidence that, at the cellular level, the mouse Hcrt/Orx system is composed of two classes of neurons with different firing properties, morphologies and synaptic input organization.
</div>
<div id="bib_Schone2011" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Schone2011,
  author = {Sch&ouml;ne, Cornelia and Venner, Anne and Knowles, David A. and Karnani, Mahesh M and Burdakov, Denis},
  title = {Dichotomous cellular properties of mouse orexin/hypocretin neurons.},
  journal = {The Journal of Physiology},
  year = {2011},
  volume = {589},
  number = {Pt 11},
  pages = {2767--79},
  url = {http://jp.physoc.org/content/early/2011/04/11/jphysiol.2011.208637.abstract},
  doi = {10.1113/jphysiol.2011.208637}
}
</pre>
</div>
</li><li><div id="Movassagh2011a"><em>Movassagh M, Choy M-K, Knowles DA, Cordeddu L, Haider S, Down T, Siggens L, Vujic A, Simeoni I, Penkett C, Goddard M, Lio P, Bennett M and Foo R</em> (2011), <strong>"Distinct Epigenomic Features in End-Stage Failing Human Hearts"</strong>, Circulation, American Heart Association.  Vol. 135</div>
<div>[<a data-toggle="collapse" href="#abs_Movassagh2011a">Abstract</a>][<a href="#bib_Movassagh2011a" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1161/CIRCULATIONAHA.111.040071" target="_blank">DOI</a>] [<a href="http://circ.ahajournals.org/content/early/2011/10/24/CIRCULATIONAHA.111.040071.abstract" target="_blank">URL</a>]</div>

<div id="abs_Movassagh2011a" class="collapse">
	<b>Abstract</b>: BACKGROUND:<br>The epigenome refers to marks on the genome, including DNA methylation and histone modifications, that regulate the expression of underlying genes. A consistent profile of gene expression changes in end-stage cardiomyopathy led us to hypothesize that distinct global patterns of the epigenome may also exist.<p>METHODS AND RESULTS:<br>We constructed genome-wide maps of DNA methylation and histone-3 lysine-36 trimethylation (H3K36me3) enrichment for cardiomyopathic and normal human hearts. More than 506 Mb sequences per library were generated by high-throughput sequencing, allowing us to assign methylation scores to â‰ˆ28 million CG dinucleotides in the human genome. DNA methylation was significantly different in promoter CpG islands, intragenic CpG islands, gene bodies, and H3K36me3-enriched regions of the genome. DNA methylation differences were present in promoters of upregulated genes but not downregulated genes. H3K36me3 enrichment itself was also significantly different in coding regions of the genome. Specifically, abundance of RNA transcripts encoded by the DUX4 locus correlated to differential DNA methylation and H3K36me3 enrichment. In vitro, Dux gene expression was responsive to a specific inhibitor of DNA methyltransferase, and Dux siRNA knockdown led to reduced cell viability.<p>CONCLUSIONS:<br>Distinct epigenomic patterns exist in important DNA elements of the cardiac genome in human end-stage cardiomyopathy. The epigenome may control the expression of local or distal genes with critical functions in myocardial stress response. If epigenomic patterns track with disease progression, assays for the epigenome may be useful for assessing prognosis in heart failure. Further studies are needed to determine whether and how the epigenome contributes to the development of cardiomyopathy.
</div>
<div id="bib_Movassagh2011a" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Movassagh2011a,
  author = {Movassagh, Mehregan and Choy, Mun-Kit and Knowles, David A and Cordeddu, Lina and Haider, Syed and Down, Thomas and Siggens, Lee and Vujic, Ana and Simeoni, Ilenia and Penkett, Chris and Goddard, Martin and Lio, Pietro and Bennett, Martin and Foo, Roger},
  title = {Distinct Epigenomic Features in End-Stage Failing Human Hearts},
  journal = {Circulation, American Heart Association},
  year = {2011},
  volume = {135},
  url = {http://circ.ahajournals.org/content/early/2011/10/24/CIRCULATIONAHA.111.040071.abstract},
  doi = {10.1161/CIRCULATIONAHA.111.040071}
}
</pre>
</div>
</li><li><div id="Glass2010"><em>Glass D, Parts L, Knowles D, Aviv A and Spector TD</em> (2010), <strong>"No correlation between childhood maltreatment and telomere length."</strong>, Biological psychiatry.  Vol. 68(6), pp. e21-2.</div>
<div>[<a href="#bib_Glass2010" data-toggle="collapse">BibTeX</a>] [<a href="https://doi.org/10.1016/j.biopsych.2010.02.026" target="_blank">DOI</a>] [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2930212/" target="_blank">URL</a>]</div>

<div id="bib_Glass2010" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Glass2010,
  author = {Glass, Daniel and Parts, Leopold and Knowles, David and Aviv, Abraham and Spector, Tim D},
  title = {No correlation between childhood maltreatment and telomere length.},
  journal = {Biological psychiatry},
  year = {2010},
  volume = {68},
  number = {6},
  pages = {e21--2},
  url = {http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2930212/},
  doi = {10.1016/j.biopsych.2010.02.026}
}
</pre>
</div>
</li><li><div id="Knowles2009pyroseq"><em>Knowles DA and Holmes S</em> (2009), <strong>"Statistical tools for ultra-deep pyrosequencing of fast evolving viruses"</strong>, NIPS Workshop: Computational Biology. , pp. 1-9.</div>
<div>[<a data-toggle="collapse" href="#abs_Knowles2009pyroseq">Abstract</a>][<a href="#bib_Knowles2009pyroseq" data-toggle="collapse">BibTeX</a>] [<a href="http://mlg.eng.cam.ac.uk/pub/pdf/KnoHol09.pdf" target="_blank">URL</a>] [<a href="http://videolectures.net/site/normal_dl/tag=62979/nipsworkshops09_knowles_smud_01.pdf" target="_blank">Slides</a>]</div>

<div id="abs_Knowles2009pyroseq" class="collapse">
	<b>Abstract</b>: We aim to detect minor variant Hepatitis B viruses (HBV) in 38 pyrosequencing samples from infected individuals. Errors involved in the amplification and ultra deep pyrosequencing (UDPS) of these samples are characterised using HBV plasmid controls. Homopolymeric regions and quality scores are found to be significant covariates in determining insertion and deletion (indel) error rates, but not mismatch rates which depend on the nucleotide transition matrix. This knowledge is used to derive two methods for classifying genuine mutations: a hypothesis testing framework and a mixture model. Using an approximate "ground truth" from a limiting dilution Sanger sequencing run, these methods are shown to outperform the naive percentage threshold approach. The possibility of early stage PCR errors becoming significant is investigated by simulation, which underlines the importance of the initial copy number.
</div>
<div id="bib_Knowles2009pyroseq" class="collapse">
<b>BibTeX</b>:
<pre>
@article{Knowles2009pyroseq,
  author = {Knowles, David A. and Holmes, Susan},
  title = {Statistical tools for ultra-deep pyrosequencing of fast evolving viruses},
  journal = {NIPS Workshop: Computational Biology},
  year = {2009},
  pages = {1--9},
  url = {http://mlg.eng.cam.ac.uk/pub/pdf/KnoHol09.pdf}
}
</pre>
</div>
</li></ol>